### DISCUSSION TITLE
Help me please

### DISCUSSION THREAD

**Discussion by rgianmaria** [community/community#66445] [Code Security]
GitHub auto create reps and i have malware in every. Devices

**Comment by Vijaykv5** [comment]
Do these steps if you got malwares while creating repos
1. Isolate the Infected Devices:
Disconnect the infected devices from the internet and any network to prevent further spread of malware.
Turn off affected devices and do not use them until they have been cleaned.

2. Scan for Malware:
Run a reputable antivirus or anti-malware scan on all of your devices to identify and remove any malicious software.
Make sure your antivirus software is up to date.

3. Change Passwords:
Change the passwords for your GitHub account, email accounts, and any other accounts that may be connected to your GitHub repositories.
Use strong, unique passwords for each account.

4. GitHub Repositories:
Review your GitHub repositories for any unauthorized changes or suspicious activities.
If you find any compromised repositories, consider archiving or deleting them if they cannot be secured.

5. Enable Two-Factor Authentication (2FA):
Enable 2FA for your GitHub account to add an extra layer of security.

6. Check Access Permissions:
Review and update the access permissions for your GitHub repositories. Make sure only trusted collaborators have access.
Revoke access for any unknown or suspicious users.

**Comment by jyotiska222** [comment]
If GitHub is auto-creating repositories under your account and you believe **malware is affecting all your devices**, it's a serious concern. Here's what you should do:

---

### 🚨 **Immediate Steps to Take:**

1. **Secure Your GitHub Account**
   - Change your GitHub password **immediately**.
   - Enable **2-Factor Authentication (2FA)** from your GitHub security settings.
   - Revoke any suspicious **OAuth apps or tokens** under **Settings > Developer Settings > Personal Access Tokens / OAuth Apps**.

2. **Audit Your Repositories**
   - Go to your GitHub profile and manually delete any repositories you did not create.
   - Review your **commit history and collaborators** for any strange activity.

3. **Scan All Your Devices**
   - Run a full system malware scan on **each device** using trusted antivirus software (like Malwarebytes, Windows Defender, or Bitdefender).
   - If possible, boot into **Safe Mode** and run the scan.

4. **Reset or Reformat (If Necessary)**
   - If infections are severe and persist, consider backing up your files and doing a **clean OS reinstall**.

5. **Check Browser Extensions**
   - Remove any unknown or untrusted browser extensions—they can steal credentials or perform malicious actions.

6. **Inspect SSH Keys**
   - Go to **GitHub > Settings > SSH and GPG Keys** and remove any SSH keys you didn’t add yourself.

---

### 🛡️ Extra Tips:
- Avoid installing pirated or cracked software—it often includes hidden malware.
- Don’t reuse passwords across different websites.
- Use a password manager for stronger security.

---

✅ *If this helps, please accept this answer to help others stay protected too!*
